MOxUnit/moxunit_runtests.m

```diff
@@ -142,3 +156,28 @@
         filenames={pwd()};
     end
 
+
+function write_xml(fname, test_result)
+
+    if isempty(fname);
+        % No file to write to, so stop now.
```

Comment:
It is just because I had the default junitxml input as `''`, so empty is the flag for no output.

However the if catch is also handled when calling `write_xml`, which is unnecessary duplication. So this catch can indeed be removed and potentially be replaced with an error.